src/Modules/Dashboard/dashbord.css

```diff
@@ -20,6 +20,10 @@
 }
 
 
+
+
+
+
 /* >>>>>>>>>>>>>>>>>>>-section project task->>>>>>>>>>>>>>>>>>> */
 
 .section-bar {
```

Comment:
We shouldn't be using normal css files, instead we should use css modules to make our styles scoped to its component. Using normal css will make more naming conflicts when the application size grows.

Reply:
should we use component.module.css for each component even the needed css styling are littel changes? this will create a lot of files
and when to use app.css or index.css?

Reply:
We can use app.css and index.css for global styles that we need in all of our application.

For each component,  we need to scope the CSS to avoid having conflicts with other team members. As we might use the same names in different files and this css will be global and will affect all of our projects and will make it difficult to reason about our css.